ProcessMaker/Events/UserUpdated.php

```diff
@@ -13,15 +14,23 @@ class UserUpdated implements SecurityLogEventInterface
     use FormatSecurityLogChanges;
 
     private User $user;
+    private array $changes;
+    private array $original;
 
     /**
      * Create a new event instance.
      *
      * @return void
      */
-    public function __construct(User $user)
+    public function __construct(User $user, array $changes, array $original)
     {
         $this->user = $user;
+        $this->changes = $changes;
+        $this->original = $original;
+        unset($this->changes['meta']);
```

Comment:
Please define a list of Key to remove then remove using:
```
$this->changes = array_diff_key($changes, array_flip($this::REMOVE_KEYS));
$this->original = array_diff_key($original, array_flip($this::REMOVE_KEYS));
```

Reply:
Solved